dotnet/src/SemanticKernel/AI/OpenAI/Services/AzureTextCompletion.cs

```diff
@@ -14,7 +15,7 @@ namespace Microsoft.SemanticKernel.AI.OpenAI.Services;
 /// <summary>
 /// Azure OpenAI text completion client.
 /// </summary>
-public sealed class AzureTextCompletion : AzureOpenAIClientAbstract, ITextCompletionClient
+public sealed class AzureTextCompletion : AzureOpenAIClientAbstract, ITextCompletion
```

Comment:
How is it that AzureTextCompletion is under "Services", but it "is a" AzureOpenAIClientAbstract which is under "Clients"?

Reply:
I know, there's a lot of code to clean up and we were planning to do that in a separate PR. We need to reorg all the code under AI. Thoughts? If we do it now the PR will grow and probably delay 2 days